code/style.css

```diff
@@ -1,4 +1,259 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* Mobile first*/
+body {
+  max-width: 1920;
+  padding: 10%;
+  
 }
+
+.full-container {
+  margin: 0;
+  padding: 10px 0px 10px 0px;
+}
+
+.navbar {
+width: 100%;
+overflow: auto;
+display: flex;
+}
+
+.navbar a {
+  font-size: 15px;
+  font-weight: bold;
+  color: black;
+  padding: 10px;
+  text-decoration: none;
+  overflow: hidden;
+  cursor: pointer;
+}
+
+.navbar a:hover {
+  background-color: #1F7980;
+  transition: 0.2s;
+  color: white;
+}
+
+.headline-section-grid {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 10px;
+  background: white;
+}
+
+.header-grid-container {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 10px;
+}
+
+.title-grid {
+  grid-column: span 3;
+  text-align: center;
+  display:inline-flex;
+  padding-top: 20px;
+}
+
+.bignews-grid1 {
+  grid-column: span 1;
+  width: 100%;
+}
+
+.bignews-grid2 {
+  grid-column: span 1;
+  display: block;
+  width: 100%;
+}
+
+.body-section-grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 15px;
+  background: white;
+}
+
+.smallnews-grid {
+  grid-column: span 4;
+  display: block;
+  width: 100%;
+  justify-content: center;
+
+}
+
+.body-section-grid img {
+  object-fit: cover;
+  border-radius: 5%;
+  height: 200px;
+  border:2px solid #fff;
+  background: url(img/tiger.png) no-repeat;
+  box-shadow: 10px 10px 5px #ccc;
+  -moz-box-shadow: 10px 10px 5px #ccc;
+  -webkit-box-shadow: 10px 10px 5px #ccc;
+  -khtml-box-shadow: 10px 10px 5px #ccc;
+  opacity: 1;
+  transition: .5 ease;
+  backface-visibility: hidden;
+
+}
+
+.cred {
+  font-size: 15px;
+}
+
+.grid-pic img {
+  animation-duration: 6s;
+  animation-iteration-count: infinite;
+  animation-name: logobounce;
+  animation-timing-function: cubic-bezier(0.280, 0.840, 0.420, 1);
+  margin: 10px;
+}
+
+@keyframes logobounce {
+  0%   { transform: scale(1,1)      translateY(0); }
+  10%  { transform: scale(1.1,.9)   translateY(0); }
+  30%  { transform: scale(.9,1.1)   translateY(-100px); }
+  50%  { transform: scale(1.05,.95) translateY(0); }
+  57%  { transform: scale(1,1)      translateY(-7px); }
+  64%  { transform: scale(1,1)      translateY(0); }
+  100% { transform: scale(1,1)      translateY(0); }
+}
+
+h1 {
+  font-size: 25px;
+  font-family: 'Averia Serif Libre';
+  font-weight: lighter;
+  letter-spacing: 3px;
+  width: 100%;
+  padding: 18px 0px 0px 10px;
+  animation-name: example;
+  animation-duration: 5s;
+  animation-iteration-count: infinite;
+}
+
+ @keyframes example {
+   0% {color: #B36A5E;}
+   25% {color: #C97C5D;}
+   50% {color: #C89F9C;}
+   75% {color: #EED7C5;}
+   100% {color: #EEE2DF;}
+ }
+
+h2 {
+  font-size: 30px;
+  padding: 2px;
+  height: 30px;
```

Comment:
Checking the page via devtool I found out a tiny bug : when you are viewing the page in mobile version the header sticks to the main banner picture (one with the boat) . So I played around and understood that it is due to the assigned hight of 30px on h2. When you just removing the line 144 : a space appear between the picture and the h2 , what makes it looks consistent with the entire layout. Also other versions (tab and desktop) are not affected with this change and all looks same. Check it out also if you like.

Reply:
Thank you for noticing, I was having issues with this but got lost along the way. This feedback really helped me and I've fixed it.